boards/arm/bt510/bt510.dts

```diff
@@ -42,37 +42,13 @@
 			gpios = <&gpio1 10 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
 			label = "Push button switch 1 (SW1)";
 		};
-	};
-
-	ids {
-		compatible = "gpio-keys";
-		id0: id_0 {
-			gpios = <&gpio1 2 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
-			label = "ID 0";
-		};
-		id1: id_1 {
-			gpios = <&gpio1 1 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
-			label = "ID 1";
-		};
-		id2: id_2 {
-			gpios = <&gpio0 25 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
-			label = "ID 2";
-		};
-	};
-
-	tm {
-		compatible = "gpio-keys";
 		tm0: tm_0 {
 			gpios = <&gpio0 3 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
 			label = "Test mode (TM)";
 		};
-	};
-
-	mag {
-		compatible = "gpio-keys";
 		mag0: mag_0 {
-			compatible = "honeywell,sm351lt";
-			gpios = <&gpio1 14 0>;
+			gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
+			label = "Honeywell sm351lt output";
```

Comment:
this is also wrong, it should be part of a `honeywell,sm351lt` compatible node, as a property.

Reply:
Oh, apologies, I did not realize that we actually had a binding and driver for this one, I'll move it to its own node.